tool/teleport/common/teleport.go

```diff
@@ -105,6 +105,8 @@ func Run(cmdlineArgs []string, testRun bool) (executedCommand string, conf *serv
 		"Specify gops addr to listen on").Hidden().StringVar(&ccf.GopsAddr)
 	start.Flag("diag-addr",
 		"Start diangonstic endpoint on this address").Hidden().StringVar(&ccf.DiagnosticAddr)
+	start.Flag("permit-user-env",
```

Comment:
I would make this a flag that accepts values with paths to read, defaulting to `~/.tsh/environment`